src/fusor/fusor.py

```diff
@@ -222,6 +224,42 @@ def assayed_fusion(
             raise FUSORParametersException(error_message) from e
         return fusion
 
+    @staticmethod
+    def internal_tandem_duplication(
+        structure: list[InternalTandemDuplicationElements],
+        causative_event: CausativeEvent | None = None,
+        assay: Assay | None = None,
+        regulatory_element: RegulatoryElement | None = None,
+        reading_frame_preserved: bool | None = None,
+        critical_functional_domains: list[FunctionalDomain] | None = None,
+        civic_molecular_profiles: list[MolecularProfile] | None = None,
+    ) -> InternalTandemDuplication:
+        """Construct an InternalTandemDuplication (ITD) object
```

Comment:
Nitpicky, but I believe this is the convention we follow (You should check to see if there are other places where a new line needs to be added)
```suggestion
        """Construct an InternalTandemDuplication (ITD) object

```